os-eap7-launch/added/launch/messaging.sh

```diff
@@ -61,11 +62,17 @@ function prepareEnv() {
 }
 
 function configure() {
+  configure_artemis_address
   inject_brokers
   configure_mq
   configure_thread_pool
 }
 
+function configure_artemis_address() {
+    IP_ADDR=${ARTEMIS_HOST:-`hostname -i`}
```

Comment:
as the ARTEMIS_HOST effectively sets the jboss.messaging.host property, it should perhaps be named JBOSS_MESSAGING_HOST ?

Reply:
agreed, thanks.